### PR TITLE
Fixed model and news tests

### PR DESCRIPTION
**Pull request by Iryna-Bzdyr**
- added permissions for category and model mutations
- fixed model and news tests

### CODE REVIEW

File: __tests__/news/news.query.test.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
 /* eslint-disable no-undef */
+/* eslint-disable no-undef */
+require('dotenv').config();
```
Comment (Alan18081) [inline __tests__/news/news.query.test.js:3]: Please, remove this line, it is going to use env config for development. Why have you added this line?

File: __tests__/user/user.mutation.test.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-undef */
+require('dotenv').config();
```
Comment (Alan18081) [inline __tests__/user/user.mutation.test.js:2]: Please, remove this line, it is going to use env config for development. Why have you added this line?

File: utils/sendGrid-email.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 const sgMail = require('@sendgrid/mail');
+require('dotenv').config();
```
Comment (Alan18081) [inline utils/sendGrid-email.js:2]: Why do you need this line?

File: __tests__/news/news.query.test.js [hunk 4]
```diff
@@ -52,14 +52,9 @@ describe('News queries tests', () => {
               }
             }
           `,
-          context: {
-            headers: {
-              token,
-            },
-          },
+          context: { headers: { token } },
```
Comment (Alan18081) [inline __tests__/news/news.query.test.js:55]: Why do you need this line if the `context` is set up when you call `setupApp`?

**Review by bandvov — APPROVED** [review]

File: __tests__/model/model.mutation.test.js [hunk 5]
```diff
@@ -1,40 +1,64 @@
 /* eslint-disable no-undef */
+require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (Iryna-Bzdyr) [reply __tests__/model/model.mutation.test.js]: Fixed

File: __tests__/model/model.mutation.test.js [hunk 6]
```diff
@@ -1,40 +1,64 @@
...
+            ... on Error {
+              message
+              statusCode
             }
           }
         }
       `,
-      variables: { category: newCategory },
+      variables: { category: newCategory, upload: '__tests__/model/img.png' },
+      context: { headers: { token } },
```
[the remark this answers is not in the document]
Comment (Iryna-Bzdyr) [reply __tests__/model/model.mutation.test.js]: Fixed

File: __tests__/model/model.mutation.test.js [hunk 7]
```diff
@@ -57,41 +81,42 @@ describe('Product queries', () => {
...
                   value
+                  lang
                 }
               }
             }
           }
         }
       `,
       variables: { model: { ...newModelMutation, category: categoryId } },
+      context: { headers: { token } },
```
[the remark this answers is not in the document]
Comment (Iryna-Bzdyr) [reply __tests__/model/model.mutation.test.js]: Fixed

File: __tests__/model/model.mutation.test.js [hunk 8]
```diff
@@ -57,41 +81,42 @@ describe('Product queries', () => {
...
     expect(model).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Мутація', lang: 'uk' },
-      { __typename: 'Language', value: 'Mutation', lang: 'en' },
+      { value: 'Мутація', lang: 'uk' },
+      { value: 'Mutation', lang: 'en' },
     ]);
     expect(model).toHaveProperty('description', [
-      { __typename: 'Language', value: 'Мутація', lang: 'uk' },
-      { __typename: 'Language', value: 'Mutation', lang: 'en' },
+      { value: 'Мутація', lang: 'uk' },
```
[the remark this answers is not in the document]
Comment (Iryna-Bzdyr) [reply __tests__/model/model.mutation.test.js]: Fixed

File: __tests__/model/model.mutation.test.js [hunk 9]
```diff
@@ -57,41 +81,42 @@ describe('Product queries', () => {
...
     });
 
     const model = createModel.data.addModel;
     modelId = model._id;
 
     expect(model).toBeDefined();
     expect(model).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Мутація', lang: 'uk' },
-      { __typename: 'Language', value: 'Mutation', lang: 'en' },
+      { value: 'Мутація', lang: 'uk' },
```
[the remark this answers is not in the document]
Comment (Iryna-Bzdyr) [reply __tests__/model/model.mutation.test.js]: Fixed

File: __tests__/model/model.query.test.js [hunk 10]
```diff
@@ -1,25 +1,38 @@
 /* eslint-disable no-undef */
+require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (Iryna-Bzdyr) [reply __tests__/model/model.query.test.js]: Fixed

File: __tests__/news/news.query.test.js (hunk 1 above)
Comment (Iryna-Bzdyr) [reply __tests__/news/news.query.test.js]: Fixed

File: __tests__/user/user.mutation.test.js (hunk 2 above)
Comment (Iryna-Bzdyr) [reply __tests__/user/user.mutation.test.js:2]: Fixed

File: utils/sendGrid-email.js (hunk 3 above)
Comment (Iryna-Bzdyr) [reply utils/sendGrid-email.js:2]: Fixed

File: __tests__/news/news.query.test.js (hunk 4 above)
Comment (Iryna-Bzdyr) [reply __tests__/news/news.query.test.js]: Fixed

**Review by ecl1pseo — APPROVED** [review]

**Review by maksym-strus — APPROVED** [review]

**Comment by Iryna-Bzdyr** [comment]
> SonarCloud Quality Gate failed.
> 
> [Bug](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=BUG)
> [Vulnerability](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=VULNERABILITY) (and [Security Hotspot](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=SECURITY_HOTSPOT) to review)
> [Code Smell](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=CODE_SMELL) [0 Code Smells](https://sonarcloud.io/project/issues?id=horondi_horondi_client_be&pullRequest=238&resolved=false&types=CODE_SMELL)
> 
> [No Coverage information](https://sonarcloud.io/component_measures?id=horondi_horondi_client_be&pullRequest=238) No Coverage information
> [](https://sonarcloud.io/component_measures?id=horondi_horondi_client_be&pullRequest=238&metric=new_duplicated_lines_density&view=list) [7.8% Duplication](https://sonarcloud.io/component_measures?id=horondi_horondi_client_be&pullRequest=238&metric=new_duplicated_lines_density&view=list)

**Pull request merged**